pkgs/top-level/perl-packages.nix

```
@@ -12909,7 +12909,7 @@ let
     outputs = [ "out" ];
     buildInputs = [ pkgs.apacheHttpd pkgs.apr pkgs.aprutil ApacheTest ExtUtilsXSBuilder ];
     propagatedBuildInputs = [ (pkgs.apacheHttpdPackages.mod_perl.override { inherit perl; }) ];
-    makeMakerFlags = "--with-apache2-src=${pkgs.apacheHttpd.dev} --with-apache2-apxs=${pkgs.apacheHttpd.dev}/bin/apxs --with-apache2-httpd=${pkgs.apacheHttpd.out}/bin/httpd --with-apr-config=${pkgs.apr.dev}/bin/apr-1-config --with-apu-config=${pkgs.aprutil.dev}/bin/apu-1-config";
+    makeMakerFlags = [ "--with-apache2-src=${pkgs.apacheHttpd.dev}" "--with-apache2-apxs=${pkgs.apacheHttpd.dev}/bin/apxs" "--with-apache2-httpd=${pkgs.apacheHttpd.out}/bin/httpd" "--with-apr-config=${pkgs.apr.dev}/bin/apr-1-config" "--with-apu-config=${pkgs.aprutil.dev}/bin/apu-1-config" ];
```

These are already barely readable now.

```suggestion
    makeMakerFlags = [
      "--with-apache2-src=${pkgs.apacheHttpd.dev}"
      "--with-apache2-apxs=${pkgs.apacheHttpd.dev}/bin/apxs"
      "--with-apache2-httpd=${pkgs.apacheHttpd.out}/bin/httpd"
      "--with-apr-config=${pkgs.apr.dev}/bin/apr-1-config"
      "--with-apu-config=${pkgs.aprutil.dev}/bin/apu-1-config"
    ];
```

---

Thanks